swim/handlers.go

```diff
@@ -48,6 +48,12 @@ type Status struct {
 	Status string `json:"status"`
 }
 
+// HealResponse contains a list of nodes where healing was attempted
+type HealResponse struct {
+	Targets []string `json:"targets"`
+	Error   error    `json:"error"`
```

Comment:
the `error` interface does not nicely serialize to json. I think it will be better to call [`Error()`](https://golang.org/pkg/builtin/#error) on the interface and put the string here.

quick test showed that an error gets serialized like this in json:
```golang
```